qt/widgets/instrumentview/src/InstrumentActor.cpp

```diff
@@ -323,132 +373,112 @@ void InstrumentActor::clearMasks() {
   }
 }
 
-Instrument_const_sptr InstrumentActor::getInstrument() const {
-  Instrument_const_sptr retval;
-
-  // Look to see if we have set the property in the properties file
-  // to define our 'default' view
-  std::string view = Mantid::Kernel::ConfigService::Instance().getString(
-      "instrument.view.geometry");
+std::vector<size_t> InstrumentActor::getMonitors() const { return m_monitors; }
 
+Instrument_const_sptr InstrumentActor::getInstrument() const {
```

Comment:
What is this still required for? Presumably there is good reason to have it.

Reply:
see `InstrumentActor::getParameterInfo`

Reply:
Ok. Maybe try and strip off the `shared_ptr` bit if you can, otherwise leave it as it is.